build.rs

```diff
@@ -1,9 +1,14 @@
+#[cfg(target_arch = "x86_64")]
 fn main() {
     let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+
     if target_arch == "x86_64" {
```

Comment:
That `if` is no longer needed, due to the config flag.

Reply:
this is still needed for cross compiling

Reply:
`target_arch` arch is not set correctly when cross compiling?